pyroengine/engine.py

```diff
@@ -26,6 +27,13 @@
 logging.basicConfig(format="%(asctime)s | %(levelname)s: %(message)s", level=logging.INFO, force=True)
 
 
+def get_folder_size(folder):
+    return (
+        sum(os.path.getsize(f) for f in glob.glob(str(folder) + "/**/*", recursive=True) if os.path.isfile(f))
+        // 1000000
+    )
+
+
```

Comment:
Let's do it more precisely : `// 1024 ** 2`
And only a suggestion but perhaps it's better to avoid creating a function for this (using the command directly in the method)

Reply:
yes good idea !